spotless.gradle

```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.diffplug.spotless'
+
+spotless {
+    java {
+        target '**/*.java'
+        googleJavaFormat().aosp()
+        removeUnusedImports()
+        trimTrailingWhitespace()
+        indentWithSpaces()
+    }
+    kotlin {
```

Comment:
I see all our new files have a copyright header. We could tell spotless to also add the header automatically if it is not present.
We could just add `licenseHeaderFile rootProject.file('spotless/license.kt')` and create the `spotless/license.kt` file.
What do you think?

Reply:
That's a good idea @aitorvs, but we already have a setting in the project that adds the header files... otherwise this would be great!